front/app/components/UI/Error/messages.ts

```diff
@@ -244,78 +244,23 @@ export default defineMessages({
     id: 'app.errors.tag_name_taken',
     defaultMessage: 'A tag with this name already exists',
   },
-  bulk_import_ideas_maximum_ideas_exceeded: {
-    id: 'app.errors.bulk_import_ideas_maximum_ideas_exceeded',
+  bulk_import_maximum_ideas_exceeded: {
+    id: 'app.errors.bulk_import_maximum_ideas_exceeded',
     defaultMessage: 'The maximum of {value} ideas has been exceeded.',
   },
-  bulk_import_ideas_idea_not_valid: {
-    id: 'app.errors.bulk_import_ideas_idea_not_valid',
-    defaultMessage: 'The resulting idea is not valid: {value}.',
-  },
-  bulk_import_ideas_blank_title: {
```

Comment:
So, some of these removed messages were never used, weren't they?

Reply:
Yes, I checked the codebase and they were no longer being thrown - because they only applied to the global importer